locale/loris.pot

```diff
@@ -213,9 +219,6 @@ msgstr ""
 msgid "Send To DCC"
 msgstr ""
 
-msgid "Sent To DCC"
-msgstr ""
-
```

Comment:
Please remember we need both "Send and Sent to DDC" in the loris.pot they are both used in instruments modules. I think to remember this comment was already made before during a previous module revision. Thanks.